doc/tagbar.txt

```diff
@@ -1079,6 +1079,15 @@ Examples:
         " Set the tag jump location to appear 25% from the top
         let g:tagbar_jump_offset = winheight(0) / 4
 <
```

Comment:
There should be a jump target line that is right justified here just before the `~` line.
```
                                                                        *g:tagbar_jump_lazy_scroll*
g:tagbar_jump_lazy_scroll~
...
```

Reply:
Done